spec.yml

```diff
@@ -0,0 +1,33 @@
+requirements:
+  base:
+    image: debian:stretch
+    pkg-manager: apt
+  fsl:
+    version: 5.0.10
+  afni:
+    version: latest
+  miniconda:
+    conda_install: [python=3.7, nipype]
+
+nfm_test:
+  data:
+    type: workflow_path
+    location: data
+  env:
```

Comment:
so this env is in addition to the requirements env, right?

Reply:
correct